src/tasks/installers/common-tasks.ts

```diff
@@ -8,6 +8,7 @@
  * SPDX-License-Identifier: EPL-2.0
  **********************************************************************/
 
+import ansi = require('ansi-colors')
```

Comment:
Can we use `import * as ansi from 'ansi-colors'` ?

Reply:
'ansi-colors' lib hasn't got d.ts file.